keps/sig-node/3759-sidecar-containers/README.md

```diff
@@ -224,9 +226,14 @@ This proposal doesn't aim to:
 The proposal is to introduce a `restartPolicy` field to Init containers and use
 it to indicate that an Init container is a sidecar container. Kubelet will start Init containers with
 `restartPolicy=Always` in the order with other Init containers, but instead of
-waiting for its completion, it will wait for the container startup completion
-(startup probe succeeded or if no startup probe defined, `postStart` handler
-completed).
+waiting for its completion, it will wait for the container startup completion.
+
+The condition for startup completion is open question. It will be either:
```

Comment:
Should this be marked as UNRESOLVED ?

When will it be resolved?

Reply:
This decision is not blocking the overall KEP from my perspective. I want to get more feedback on it before finalizing. 

My understanding is that your position is `Ready` condition would be an easy and straightforward solution, but not strongly against relying on startup probes. Please correct me if I'm wrong

Reply:
I meant that you should mark it as unresolved and indicate what / when we will resolve it.

That is my position, yes

Reply:
marked as unresolved